corehq/apps/reports/views.py

```diff
@@ -994,7 +995,7 @@ def report_notification(self):
                 instance.day = calculate_day(instance.interval, instance.day, day_change)
 
             if not self.can_edit_report(instance):
-                return HttpResponseBadRequest()
+                raise Http400()
```

Comment:
shouldn't this be a `403`?

Reply:
Maybe, but I guess I was just changing this to do what the original intention clearly was.

Reply:
was the original intention the right one, though?

Reply:
haha okay fine